cmd/build.go

```diff
@@ -99,7 +99,7 @@ If you want to push the built image to an image repository using the [--push] op
 
 	buildCmd.PersistentFlags().StringVarP(&config.tag, "tag", "t", "", "Container image name and optionally, a tag in the 'name:tag' format.")
 	buildCmd.PersistentFlags().BoolVar(&rootConfig.Buildah, "buildah", false, "Build project using buildah primitives instead of Docker.")
-	buildCmd.PersistentFlags().StringVar(&config.dockerBuildOptions, "docker-options", "", "Specify the Docker build options to use. Value must be in \"\".")
+	buildCmd.PersistentFlags().StringVar(&config.dockerBuildOptions, "docker-options", "", "Specify the Docker build options to use. Value must be in \"\". The following docker options are not allowed: '--help','-t','--tag','-f','--file'.")
```

Comment:
Minor comment but the word docker should be capitalized and I'd prefer the word "supported" rather than "allowed", as these options are valid in a Docker context just not within this '--docker-options' flag:

"The following Docker options are not supported:"

Reply:
Changes addressed